app/src/components/main/grid/gridBody/nestedGridRow/nestedGridRow.jsx

```diff
@@ -108,10 +108,38 @@ export const NestedGridRow = ({
   const showPreviousButton = isLoadPreviousButtonVisible(page);
   const showMoreButton = isLoadMoreButtonVisible(page);
   const showLoadCurrentStepButton = isLoadCurrentStepButtonVisible(page, data, allNestedStepsData);
+  const [firstRowId, setFirstRow] = React.useState(null);
+  const [direction, setDirection] = React.useState(null);
+  const itemIntoViewRef = React.useRef(null);
+
+  const calculateFirstRowIdOnLoadPreviousClick = (contentItems) => {
+    const isStep = (item) => 'hasContent' in item;
+    setFirstRow((contentItems.find((item) => !isStep(item)) || {}).id);
```

Comment:
What potential situation when we're don't find log and fallback to {}?

Reply:
For now I can not, imagine the situation, when we did not find id.
But this situations always accurse for me, when I developed, because was situation when we have nested step, and no logs inside, but have button(load prev).  I know, that situation like this, probably never accurse. 

As for me, I think that is better to have fallback, because we cannot cover all cases when we can did not find id.
But if this situation accurse we protect our self from app crashing.